.github/workflows/ci.yml

```
@@ -45,15 +45,16 @@ jobs:
     strategy:
       matrix:
         images:
-          - localRegistry: localhost:5000/ketch-com/go-ketch-forwarder
-            githubRegistry: ghcr.io/ketch-com/go-ketch-forwarder
+          - githubRegistry: ghcr.io/ketch-com/go-ketch-forwarder
             # ecrRepository: go-ketch-forwarder
+            localRegistry: localhost:5000/ketch-com/go-ketch-forwarder
             dockerfile: docker/ketch-event-forwarder/Dockerfile
       fail-fast: false
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     permissions:
-      contents: read
+      contents: write
```

```suggestion
      contents: read
```
IIRC, `contents: write` means you are committing to this repository. We don't need that permission.